File: src/app/gkill/api/req_res/get_gps_log_request.go
// ˅
package req_res

import "time"

// ˄

type GetGPSLogRequest struct {
	// ˅

	// ˄

	SessionID string `json:"session_id"`

	StartDate time.Time `json:"start_date"`

	EndDate time.Time `json:"end_date"`

	// ˅

	// ˄
}

// ˅

// ˄

File: src/client/classes/api/req_res/get-gps-log-request.ts
'use strict'

import { GkillAPIRequest } from '../gkill-api-request'

export class GetGPSLogRequest extends GkillAPIRequest {

    start_date: Date

    end_date: Date

    constructor() {
        super()
        this.start_date = new Date(0)
        this.end_date = new Date(0)
    }

}


